docs/book/src/reference/envtest.md

```diff
@@ -19,6 +19,14 @@ export K8S_VERSION=1.21.2
 curl -sSLo envtest-bins.tar.gz "https://go.kubebuilder.io/test-tools/${K8S_VERSION}/$(go env GOOS)/$(go env GOARCH)"
 ```
 
+However, in the case of M1 Mac using AppleSillicon, the arm64 binary is not yet available, so the test in the Makefile must be changed.
+```makefile
+test: manifests generate fmt vet envtest ## Run tests.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
```

Comment:
is the amd64 working in the AppleSillicon?

Reply:
As you say, there is no kubebuilder-tools supporting darwin/arm64, so using this flag will allow M1Mac to run binaries for intel via Rosseta. (Temporary solution)

Reply:
We have them now :-) 
With the latest release : https://storage.googleapis.com/kubebuilder-tools

See for 1.23.5

Reply:
Hi @shintard, 

I checked all that is missing for we support the Apple Silicon and I updated the comment: https://github.com/kubernetes-sigs/kubebuilder/issues/1932#issuecomment-1112564792